pyfolio/timeseries.py

```diff
@@ -512,7 +516,8 @@ def calc_multifactor(returns, factors):
     return results.params
 
 
-def rolling_beta(returns, benchmark_rets, rolling_window=63):
+def rolling_beta(returns, benchmark_rets,
```

Comment:
Agreed. I changed it to `factor_returns` as the functions are unaware of which factor gets passed in. For that reason, I left `tears.py` to still use `benchmark_rets` as there is specific meaning associated. Make sense?